lib/protoparser/prometheus/proto.go

```diff
@@ -0,0 +1,838 @@
+package prometheus
```

Comment:
This file has unused unmarshal method, it looks like a bug to me.

Reply:
added support of a whole prometheus protobuf, but parsing only a subset at the moment, histograms will be supported later and these unused methods will be used later